src/omemo/omemo.c

```diff
@@ -543,7 +543,7 @@ omemo_prekeys(GList** prekeys, GList** ids, GList** lengths)
 void
 omemo_set_device_list(const char* const from, GList* device_list)
 {
-    log_debug("[OMEMO] Setting device list for %s", from);
+    log_debug("[OMEMO] Setting device list for %s", (from == NULL) ? "" : from);
```

Comment:
Hmm, this is a semantic change and more than originally intended by my commit :)

Another possibility here would be to move the `log_debug()` statement into the else clause, then the macro wouldn't be needed at all since `from` cannot be NULL. And the `log_debug()` didn't make much sense if `from == NULL`, or?